### PR TITLE
[web-server] Modify audience and scope fields in mozilla-auth0

### PR DESCRIPTION
**Pull request by helfi92**
Relates to https://github.com/taskcluster/taskcluster/issues/806.

### CODE REVIEW

File: services/web-server/README.md [hunk 1]
```diff
@@ -120,8 +120,7 @@ UI_LOGIN_STRATEGIES='{
         "domain": "<auth0 subdomain>",
         "clientId": "<clientId from registration of client>",
         "clientSecret": "<clientSecret from registration of client>",
-        "audience": "api.sso.mozilla.com",
-        "scope": "display:all taskcluster-credentials openid profile"
+        "scope": "openid profile"
```
Comment (djmitche) [inline services/web-server/README.md]: There's no other acceptable value for this field, so I don't think it should be exposed as configuration.

File: services/web-server/src/login/strategies/mozilla-auth0.js [hunk 2]
```diff
@@ -201,7 +200,6 @@ module.exports = class MozillaAuth0 {
           domain: strategyCfg.domain,
           clientID: strategyCfg.clientId,
           clientSecret: strategyCfg.clientSecret,
-          audience: strategyCfg.audience,
           scope: strategyCfg.scope,
```
Comment (djmitche) [inline services/web-server/src/login/strategies/mozilla-auth0.js]: `'openid profile'` can be hard-coded here.

File: services/web-server/README.md (hunk 1 above)
Comment (helfi92) [reply services/web-server/README.md]: True that!

File: services/web-server/test/strategyMozillaAuth0_test.js [hunk 3]
```diff
@@ -10,7 +10,6 @@ suite(testing.suiteName(), () => {
         strategies: {
           'mozilla-auth0': {
             domain: 'login-test.taskcluster.net',
-            audience: 'login-test.taskcluster.net',
             clientId: 'abcd',
             clientSecret: 'efgh',
             scope: 'taskcluster',
```
Comment (djmitche) [inline services/web-server/test/strategyMozillaAuth0_test.js]: ..and here

File: services/web-server/README.md (hunk 1 above)
Comment (djmitche) [reply services/web-server/README.md]: still needs to be removed here..

**Pull request merged**